src/cli.ts

```diff
@@ -307,7 +331,11 @@ function logAddressInfo(ctx: Context) {
   log(`secp256k1 public key: 0x${compressedPubKey}`)
 }
 
-async function logBalanceInfo(ctx: Context) {
+/**
+ * @description Returns the balance info of the account

```

Comment:
you should put function description in @description and returns summary in @returns

Reply:
It does not return anything